cmd/app/add_test.go

```diff
@@ -352,7 +352,7 @@ func TestAppAddCommand(t *testing.T) {
 				mockProjectConfig.On("Cache").Return(mockProjectCache)
 				cm.Config.ProjectConfig = mockProjectConfig
 			},
-			ExpectedAsserts: func(t *testing.T, cm *shared.ClientsMock) {
+			ExpectedAsserts: func(t *testing.T, ctx context.Context, cm *shared.ClientsMock) {
```

Comment:
note: Every reference to `ExpectedAsserts: func(...)` had to be updated to include `ctx`. 🔁

Reply:
Another tedious change - thanks for rolling up the sleeves - but this matches actual command execution so much better 🙏 ✨